ludwig/modules/tabnet_modules.py

```diff
@@ -92,6 +90,11 @@ def __init__(
             output_size,
             output_size)
 
+        # Assign tensors to be used in forward pass
```

Comment:
Could we extend this comment with a higher-level statement about about why register_buffer is necessary for GPU training for these tensors specifically? Or perhaps a link to where this suggestion comes from?

Reply:
Done.